customerio.go

```diff
@@ -19,35 +19,19 @@ type CustomerIO struct {
 	SSL    bool
 }
 
-// CustomerIOError is returned by any method that fails at the API level
-type CustomerIOError struct {
-	status int
-	url    string
-	body   []byte
-}
-
-func (e *CustomerIOError) Error() string {
-	return fmt.Sprintf("%v: %v %v", e.status, e.url, string(e.body))
-}
-
 // NewCustomerIO creates a new CustomerIO object to perform requests on the supplied credentials
 func NewCustomerIO(siteID, apiKey string) *CustomerIO {
 	return &CustomerIO{siteID, apiKey, "track.customer.io", true}
 }
 
 // Identify identifies a customer and sets their attributes
 func (c *CustomerIO) Identify(customerID string, attributes map[string]interface{}) error {
-	j, err := json.Marshal(attributes)
-
-	if err != nil {
-		return err
-	}
-
-	status, responseBody, err := c.request("PUT", c.customerURL(customerID), j)
+	attrBytes, err := json.Marshal(attributes)
```

Comment:
Renaming the variable for encoded json values from `j` to something more descriptive is fine. However if that's the route we go would be better to use something consistent.

I see you have `attrBytes` here and in other places went with just `byts`. One option I've used before is to refer to json marshaled values used in API calls as `marshaledBody`

Reply:
@nafisfaysal nice point. Yeah, I rename it because of the `json.Marshal`  return `[ ]bytes` but I think you `marshaledBody` I was a good name. Maybe I can just keep it `marshaledB` :smiley: